### PR TITLE
Ba 63 Book Details added

### PR DESCRIPTION
**Pull request by Nikita5796**

### CODE REVIEW

File: src/app/book/booklist/booklist.component.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <tr>
+      <th>Book Image</th>
+      <th>Book Title</th>
+      <th>Book Price</th>
+      <th>Book Rating</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr *ngFor = "let book of books" >
+      <!-- <td>
```
Comment (bala0105) [inline src/app/book/booklist/booklist.component.html]: remove commented code

**Comment by ketannaik** [comment]
This PR is for Book details right??
Then why is BookList code added in it??

Did you create this branch before merging the BA-62 into master??

**Review by ketannaik — CHANGES_REQUESTED** [review]
This PR is for Book details right??
Then why is BookList code added in it??

Did you create this branch before merging the Ba

File: src/app/book/booklist/booklist.component.html (hunk 1 above)
Comment (Nikita5796) [reply src/app/book/booklist/booklist.component.html]: done. yes I have created branch before merging. and I just add router link to get bookid with route which is in booklist component to pass it in bookdetails component. ('book/{{ book.bookId }}') thats why changes made in booklist.html.

**Pull request merged**